src/styles/blocks.scss/header-nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    display: flex;
```

Comment:
The `display: flex;` property is not necessary here since you are not using any of the flexbox properties on this element. It's better to remove it to keep the code clean.